### ISSUE TITLE
Add @include with nested selectors to Outline View

### ISSUE THREAD

**Issue by paulozoom** [microsoft/vscode-css-languageservice#193]
It’s extremely common for SCSS developers to use `@include` as shortcut for `@media` queries or often-repeated selectors, .e.g.:

```scss
@include tablet {
  .box {
  }
}

@include dark-mode {
  .menu {
  }
}
```

However, selectors nested inside `@include` don’t appear nested in the Outline View, as they would if nested in `@media` since cdf495432eb8c1cba25e9d83b94895305c757cd1. It’d be extremely useful to see these in the outline as well.

**Comment by aeschli** [comment]
![image](https://user-images.githubusercontent.com/6461412/98863922-61729f80-2469-11eb-83af-319fa11b7596.png)

The nested selector is in the outline, but not a node for `@include`

Are you suggesting a node for  `@include` if it has a body as argument?

**Comment by paulozoom** [comment]
Yes, precisely, if it has a body, would count as a note for navigation and symbol tree purposes.
